src/core/server/queue/tasks/mailer/content.ts

```diff
@@ -33,7 +33,7 @@ function render(env: Environment, { name, context }: EmailTemplate) {
           return reject(err);
         }
 
-        return resolve(html);
```

Comment:
If `html` can be undefined or null, add a guard clause here instead of asserting `html`.

Reply:
I think `html` is only `null` when `err` is not `null`, but I'll add a guard.